docs/support/support_system.md

```diff
@@ -0,0 +1,164 @@
+# Support System
+## Why set up a Support System
+CommCare is a powerful technology that caters to programs of different nature from simple data collection from a few hundred users to large scale deployment of millions of users. As with any technology, users using CommCare will need support beyond the user documentation to be successful at their job. To this extent, setting up a robust support system becomes essential for the success of any project in which CommCare is set.
+
+### Identifying stakeholders
+Projects using CommCare often have stakeholders who use and interact with CommCareHQ in different ways and they often have varied levels of technical expertise. Some examples of the stakeholders are community health / frontline workers (CHWs / FLWs), supervisors who monitor the FLWs, Monitoring and Evaluation (M&E) staff and data teams that deal with the data analytics and reporting side of the program, project team that is managing the CommCareHQ project setup and high level program staff. These users interact with CommCare mobile and server in different ways and will need different types of support.
+
+Below are some example support issues that might come up from various stakeholders of the program
+- An FLW not being able to sync or submit data to CommCareHQ server
+- A project team member not seeing upto data on system
+
+When setting up a support system we recommend you to do an exercise of identifying the stakeholders in your program and think through the program/technical workflows that these stakeholders would be part of and the potential issues that they might encounter.
+
+## Components of a Support System
+A good support system should have the following components:
+- Training and documentation with common support questions and issues for end users
+- Support channels for users to get the support they need
+- Support team members
+- Support processes and tools to track, triage, troubleshoot and escalate incoming issues
+
+Below we give details on each component.
+
+### Training and Documentation
+A major portion of support issues end up being simply queries from users who may not know how to perform an action. The best way to avoid these is by providing training at the time of onboarding the users and documentation on how to use the system. 
+
+[//]: # (<Todo; link to example collateral docs on how to use the system>)
```

Comment:
Is [this](https://confluence.dimagi.com/display/commcarepublic/Home) a good link to use here?